static/flow/views/program-editor-panel.js

```diff
@@ -9,13 +9,13 @@ var ProgramEditorPanel = function(options) {
     this.m_diagram      = null;
     this.m_diagramName  = null;
     this.m_diagramDisplayedName  = null;
-    this.m_svgDrawer    = null;
```

Comment:
I also hate this weird spacing on assignments.  It's all over at the top of modules.  We should add a story to pull it out.  Where's the wrecking ball?